_translator-files/po/es/Getting-Started.po

```diff
@@ -180,64 +171,48 @@ msgstr "Rellena al menos “Alias/Nombre” y cierra la ventana."
+msgstr "Antes de conectarte a un Servidor **no** deberías de poder escucharte.  Escuchar tu sonido desde Jamulus (y no tu audio directo) se denomina “La Regla de Oro” y te permite tocar en sincronización con otras personas ([ver las FAQ](/wiki/FAQ#por-qué-no-debería-escuchar-mi-propia-señal))."
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:62
-#, fuzzy, no-wrap
-#| msgid "**Before playing with others, we recommend that you connect to an empty server to test your setup**, and make sure you are listening to the signal coming back from the server (if possible) and not yourself.\n"
+#, no-wrap
 msgid "**Before playing with others, we recommend that you connect to an empty Server to test your setup**, and make sure you are listening to the signal coming back from the Server (if possible) and not yourself.\n"
-msgstr "**Antes de tocar con otros, recomendamos que te conectes a un servidor vacío para probar tu configuración**, y asegurarte de que escuchas la señal que vuelve del servidor (si es posible) y no tu audio directo.\n"
+msgstr "**Antes de tocar con otros, recomendamos que te conectes a un Servidor vacío para probar tu configuración**, y que te asegures de que escuchas la señal que vuelve del Servidor (si es posible) y no tu audio directo.\n"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:64
-#, fuzzy
-#| msgid "Now use the “Connect” button in the Jamulus main window to join a server. A window will now open:"
 msgid "Now use the “Connect” button in the Jamulus main window to join a Server. A window will now open:"
-msgstr "Ahora usa el botón de “Conectar” en la ventana principal de Jamulus para unirte a un servidor. Se abrirá una ventana:"
+msgstr "Ahora usa el botón de “Conectar” en la ventana principal de Jamulus para unirte a un Servidor. Se abrirá una ventana:"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:69
-#, fuzzy, no-wrap
-#| msgid ""
-#| "<figure>\n"
-#| "  <img src=\"{% include img/en-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot of the connection window\" style=\"border: 5px solid grey;\">\n"
-#| "  <figcaption>Connect to a server</figcaption>\n"
-#| "</figure>\n"
+#, no-wrap
 msgid ""
 "<figure>\n"
 "  <img src=\"{% include img/en-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot of the connection window\" style=\"border: 5px solid grey;\">\n"
 "  <figcaption>Connect to a Server</figcaption>\n"
 "</figure>\n"
 msgstr ""
 "<figure>\n"
-"  <img src=\"{% include img/es-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
-"  <figcaption>Conéctate a un servidor</figcaption>\n"
+"  <img src=\"{% include img/es-screenshots/connection-setup-window.inc %}\" loading=\"lazy\" alt=\"Screenshot of the connection window\" style=\"border: 5px solid grey;\">\n"
```

Comment:
Shouldn't the screenshot alt text get translated?

Reply:
For some reason I thought that text wasn't shown. Done.